react-native/architecture/components/my-component/component.s.md

```diff
@@ -0,0 +1,16 @@
+# [Standard] Components folder - Component file
+
+## Standard
+
+- The file name should be `MyComponent.style.js`
+- Exports the component class by default
+
+  > **Why?:** Linked to code splitting and single responsability. Facilitates code testing.
+
+- Contains a unique `render()`
```

Comment:
Can a file contains multiple subcomponents. With `styled-component`, this is pretty standard:

```js
import styled from "styled-component/native";

const HeaderContainer = styled.View`
  flex: 1;
  background-color: "red";
`

const HeaderText = styled.Text`
  color: "cyan";
  font-size: 50;
`

const Header = () => (
  <HeaderContainer>
     <HeaderText> YOLO </HeaderText>
  </HeaderContainer>
);
export default Header;
```

Reply:
Not our standard yet